### PR TITLE
DUI: Implement ElementTypeToShortConverter

### PR DESCRIPTION
**Pull request by aosyatnik**
`ElementTypeToShortConverter` shows correct label for every type of element, when mouse is over.
Result:
image
image

Reviewers
@Benglin , please take a look.

Link to YouTrack:
[MAGN-6197](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6197) DUI: Implement ElementTypeToShorthandConverter

### CODE REVIEW

File: src/DynamoCoreWpf/Views/Search/LibraryView.xaml [hunk 1]
```diff
@@ -195,25 +195,67 @@
...
+                               Text="{Binding Path=ElementType,Converter={StaticResource ElementTypeToShortConverter}}"
+                               VerticalAlignment="Center"
+                               HorizontalAlignment="Right"
+                               TextBlock.FontSize="10"
+                               TextBlock.Foreground="{StaticResource NestedMemberTextColor}"
+                               Margin="0,0,10,0">
+                        <TextBlock.Style>
+                            <Style TargetType="{x:Type TextBlock}">
+                                <Setter Property="Visibility"
+                                        Value="Hidden" />
```
Comment (Benglin) [inline src/DynamoCoreWpf/Views/Search/LibraryView.xaml]: I think you can fold `Visibility="Hidden"` into the `TextBlock` itself without this additional `Style.Setter`.

File: test/DynamoCoreTests/ConverterTests.cs [hunk 2]
```diff
@@ -491,5 +491,31 @@ public void SelectedItemToActiveConverterTest()
...
+
+            result = converter.Convert(ElementTypes.CustomNode, null, null, null);
+            Assert.AreEqual("DS", result);
+
+            result = converter.Convert(ElementTypes.BuiltIn, null, null, null);
+            Assert.AreEqual(String.Empty, result);
+
+            result = converter.Convert(ElementTypes.None, null, null, null);
+            Assert.AreEqual(String.Empty, result);
+        }
```
Comment (Benglin) [inline test/DynamoCoreTests/ConverterTests.cs]: Thanks for adding this test.

**Comment by Benglin** [comment]
Hi @aosyatnik, minor tweaks and then we're good to go! Thanks.

**Comment by aosyatnik** [comment]
Wow, something went really wrong... I think I'll reopen this PR, made on the newest master.

**Pull request closed without merging**